### PR TITLE
Add favicons

### PR DESCRIPTION
**Pull request by ohbarye**
## Issue

I noticed your website is, unfortunately, missing a favicon.

image

## Change

Added a favicon and web clip icons for some platforms. How do you feel about them?

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,7 +12,12 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
+    <link
+      rel="icon"
+      type="image/png"
+      href="%PUBLIC_URL%/android-chrome-384x384.png"
+    />
```
Comment (kachick) [inline public/index.html:20]: I didn't know the naming rules for favicon for each devices 😮 

https://ao-system.net/favicongenerator/

**Review by kachick — APPROVED** [review]
LGTM 👍 
This is not a crucial issue for users.

But these error will unfairly lower developer's rating. 💯

**Comment by pankona** [comment]
LGTM! Thank you for your great contribution!

**Comment by ohbarye** [comment]
thx for your quick review 👍

**Pull request merged**